Makefile

```diff
@@ -4,24 +4,26 @@ ifneq ("$(wildcard docker-compose.local.yml)","")
 DOCKER_COMPOSE_YML += --file docker-compose.local.yml
 endif
 
-QUICKSTART := go
+language := node
+SUCCESS_MESSAGE := "✅ $(language) quickstart is running on http://localhost:8000"
 
 .PHONY: up
 up:
 	$(DOCKER_COMPOSE) \
 		$(DOCKER_COMPOSE_YML) \
-		$@ --detach --remove-orphans \
-		$(QUICKSTART)
+		$@ --build --detach --remove-orphans \
+		$(language)
+	@echo $(SUCCESS_MESSAGE)
 
 .PHONY: logs
 logs:
 	$(DOCKER_COMPOSE) \
 		$@ --follow \
-		$(QUICKSTART)
+		$(language)
 
 .PHONY: stop build
 stop build:
 	$(DOCKER_COMPOSE) \
 		$(DOCKER_COMPOSE_YML) \
 		$@ \
-		$(QUICKSTART)
+		$(language)
```

Comment:
using lowercase args seems like convention `make up language=node`

Reply:
I wouldn't necessarily say it's standard in Makefiles, but it's definitely easier to type, so I'd say it's a win for this project. You could even go with `lang=node` to be shorter, which has a tradeoff of being a little less clear.